ansible/roles/test/files/ptftests/pfc_wd.py

```diff
@@ -45,7 +47,7 @@ def runTest(self):
         for match in matches:
             for port in match.split():
                 dst_port_list.append(int(port))
-        src_mac = self.dataplane.get_mac(0, 0)
+        src_mac = self.dataplane.get_mac(*random.choice(self.dataplane.ports.keys()))
```

Comment:
will this cause any issues on Mellanox platforms?

Reply:
No, this won't because this is retrieved once, and `pfc_wd.py` doesn't check traffic balancing.